app/utilities/encryption.py

```diff
@@ -37,6 +52,26 @@ def create_access_token(account):
     return encoded_jwt
 
 
+def check_jwt(token):
+    try:
+        payload = jwt.decode(
+            token, os.getenv('SECRET_KEY'),
```

Comment:
I would set these env vars in the beginning of the program. If you forget them in the env file, then you'll know right in the beginning.

Reply:
Ok will undo that!